doc/ref/plot_options/data.ipynb

```diff
@@ -45,7 +59,557 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.xarray  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "ds = hvsampledata.air_temperature(\"xarray\")\n",
+    "\n",
+    "plot1 = (ds - 273).hvplot.image(width=400, title=\"Default check for symmetry\")\n",
+    "plot2 = (ds - 273).hvplot.image(check_symmetric_max=10, width=400, title=\"Avoid symmetry check above 10\")\n",
+    "\n",
+    "plot1 + plot2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `transforms`\n",
+    "\n",
+    "The `transforms` option allows you to modify data values for specific dimensions before plotting. Unlike the [`fields`](#fields) option which only renames or adds metadata, `transforms` applies HoloViews expressions to the data. It accepts a dictionary where each key is a dimension (for example, a DataFrame column name) and each value is a HoloViews expression built with `holoviews.dim()` that defines how to transform that dimension.\n",
+    "\n",
+    "For instance, if you have a 'probability' column with values between 0 and 1 and you want to display them as percentages, you can define a transformation as:\n",
+    "\n",
+    "`percent = hv.dim('probability') * 100`\n",
+    "\n",
+    "When passed via the transforms keyword, this expression multiplies all values in the ‘probability’ column by 100 before plotting."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import holoviews as hv\n",
+    "import hvplot.pandas  # noqa\n",
+    "\n",
+    "df = pd.DataFrame({'value': np.random.randn(50), 'probability': np.random.rand(50)})\n",
+    "percent = hv.dim('probability') * 100\n",
+    "\n",
+    "df.hvplot.scatter(\n",
+    "    x='value', y='probability', transforms={'probability': percent}\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## `use_dask`\n",
+    "\n",
+    "The `use_dask` option tells hvPlot to treat your data as dask-backed, enabling out‐of‐core and parallelized computation for datasets that might not fit in memory. When set to `True`, hvPlot checks whether the provided data is a Dask DataFrame (or similar Dask object) and uses the appropriate processing branch. If you set `persist=True`, it persists the data in memory for improved performance on subsequent operations."
```

Comment:
I had a quick look at the code and think that only xarray and Intake can be affected by `use_dask`. Can you please check this further? If that's correct, only mention xarray (intake support is a bit up in the air).